docs/framework/configure-apps/file-schema/winforms/index.md

```diff
@@ -54,5 +54,4 @@ The `<System.Windows.Forms.ConfigurationSection>` element can include one or mor
 
 ## See also
 
-[Configuration File Schema](Configuration%20File%20Schema%20for%20the%20.NET%20Framework.md)   
-[High DPI Support in Windows Forms](../../../../../docs/framework/winforms/high-dpi-support.md)
+[High DPI Support in Windows Forms](../../../../../docs/framework/winforms/high-dpi-support-in-windows-forms.md)
```

Comment:
The Configuration File Schema topic should still go in See Also, since it's the parent topic: [Configuration File Schema](../index.md).